### ISSUE TITLE
Cannot use lava/water eggs in tanks

### ISSUE THREAD

**Issue by Jordyn2012** [setycz/ChickensMod#132]
#### Issue Description:
I play with a custom (1.12.2) modpack that includes the chickens mod, more chickens, hatchery, and some tech mods like ender io, tinkers, and thermal expansion. I recently noticed that I cannot get the lava eggs to go into any sort of tank as they did in 1.10.2, and cannot find any information online regarding this issue. The only thing I can do with the egg is right click the ground to make a lava source block, but the egg cannot be used as a fuel source anymore. I am not sure if this has been changed, but I was really excited to use the lava eggs as they are much easier to deal with than the lava buckets themselves.

#### Steps to reproduce (important):

1. Right click tank (ender io, tinkers, thermal expansion, etc.) with egg in hand (OR)
2. Shift-click egg into tank GUI (OR)
3. Drag and drop egg into tank GUI
...

#### What happens:
The egg will not fill the tank with lava nor will the tank accept the egg. When you right click the tank with the egg, it simply opens the GUI. When you shift click or drag the egg into a slot, the tank doesn't accept it.

#### What you expected to happen:
The tank fills with lava from the egg, as it did in 1.10.2

____
#### Affected Versions (Do *not* use "latest"):

- Chickens: 6.1.0v, 6.0.4v, 6.0.3v, 6.0.2v (all versions for 1.12.2)
- Minecraft: 1.12.2
- Forge: 14.23.4.2729

**Comment by Tazgirl18992** [comment]
I _think_ this is an issue to do with the IFluidHandlerItem not being implemented with those eggs. It seems to have been fixed in the code so we'll need to wait until the mod is updated.

If you have other mods, you can make a work around with some kind of mechanical user using the eggs with a fluid collector picking up the lava/water.

**Comment by GenDeathrow** [comment]
I'll look into it.. the fix should have been live a while back ago. but I will check it out.

**Comment by Tazgirl18992** [comment]
Thanks. I know I have the latest version of chickens mod, but putting lava/water eggs directly into tanks/machines in other mods still isn't working. Even using the the tank or machines GUI will not work.

**Comment by GenDeathrow** [comment]
yeah its not live yet.. not sure why I didn't get that build finished. Im looking at pushing out a couple of mods soon, this will be in the batch that's updated.

**Comment by Tazgirl18992** [comment]
Can't wait! Thanks for the help.

**Comment by fivelow** [comment]
Still having this issue in August of 2019. Was there any forward movement on this?
